Description: Don't use smartquotes for Sphinx
 Don't let Sphinx inconsistently alter quotes and dashes in rst files.
Author: Laszlo Boszormenyi (GCS) <gcs@debian.org>
Bug-Debian: https://bugs.debian.org/1028456
Forwarded: no
Last-Update: 2023-01-12

---


--- tiff-4.5.0.orig/doc/conf.py
+++ tiff-4.5.0/doc/conf.py
@@ -175,3 +175,5 @@ man_pages = [
 
 # If true, show URL addresses after external links.
 man_show_urls = True
+
+smartquotes = False
